src/utills/IsMobile.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-useless-escape */
+export const mobileCheck = function () {
```

Comment:
please add a link to where you copied this from for refence

Reply:
added